app.py

```diff
@@ -54,20 +56,41 @@ def _project_list_query():
     global all_entries
     global search_input
     global augur_db
+    global repo_dict
+    global org_dict
 
     # query of available orgs / repos
     logging.debug("AUGUR_ENTRY_LIST - START")
-    pr_query = f"""SELECT * FROM augur_data.explorer_entry_list"""
-
+    pr_query = f"""SELECT DISTINCT
+                        r.repo_git,
+                        r.repo_id,
+                        r.repo_name,
+                        rg.rg_name
+                    FROM
+                        repo r
+                    JOIN repo_groups rg
+                    ON rg.repo_group_id = r.repo_group_id
+                    ORDER BY rg.rg_name"""
+
+    # query for search bar entry generation
     df_search_bar = augur_db.run_query(pr_query)
 
+    # handling case sesitive options for search bar
     entries = np.concatenate((df_search_bar.rg_name.unique(), df_search_bar.repo_git.unique()), axis=None)
     entries = entries.tolist()
     entries.sort(key=lambda item: (item, len(item)))
 
+    # generating search bar entries
     lower_entries = [i.lower() for i in entries]
     all_entries = list(zip(lower_entries, entries))
 
+    # generating dictionary to be used to select repo_ids from repos selected in the search bar
+    repo_dict = df_search_bar[["repo_git", "repo_id", "repo_name"]].set_index("repo_git").T.to_dict("list")
+
+    # generating dictionary to be used to get repos from selected repo groups
+    org_dict = df_search_bar.groupby("rg_name")["repo_git"].apply(list).to_dict()
```

Comment:
that's smart. Could you please comment the desired outcome of these two processing steps above them? pandas chaining is difficult to follow but that process looks good.